var
thrown
=
new
Test262Error
(
)
;
var
caught
;
var
sameBlock
=
false
;
var
subsequentField
=
false
;
var
subsequentBlock
=
false
;
try
{
class
C
{
static
{
throw
thrown
;
sameBlock
=
true
;
}
static
x
=
subsequentField
=
true
;
static
{
subsequentBlock
=
true
;
}
}
}
catch
(
error
)
{
caught
=
error
;
}
assert
.
sameValue
(
caught
thrown
)
;
assert
.
sameValue
(
sameBlock
false
'
same
block
'
)
;
assert
.
sameValue
(
subsequentField
false
'
subsequent
field
'
)
;
assert
.
sameValue
(
subsequentBlock
false
'
subsequent
block
'
)
;
reportCompare
(
0
0
)
;
